docs/SUMMARY.md

```diff
@@ -0,0 +1,5 @@
+* [Introduction](../README.md)
+* [API](./API.md)
+* [Developer quickstart](./Quickstart.md)
```

Comment:
I think renaming this header to just `quickstart` is sufficiently understandable.